setup.py

```diff
@@ -91,19 +91,13 @@ def get_version(version_file):
         "cyhy-logging @ git+https://github.com/cisagov/cyhy-logging.git@first-commits",
         "jsonschema",
         "rich",
-        "setuptools >= 24.2.0",
+        "setuptools",
     ],
     extras_require={
         "test": [
             "coverage",
-            # coveralls 1.11.0 added a service number for calls from
-            # GitHub Actions. This caused a regression which resulted in a 422
-            # response from the coveralls API with the message:
-            # Unprocessable Entity for url: https://coveralls.io/api/v1/jobs
-            # 1.11.1 fixed this issue, but to ensure expected behavior we'll pin
-            # to never grab the regression version.
-            "coveralls != 1.11.0",
-            "docker == 7.1.0",
+            "coveralls ",
```

Comment:
I'm not sure why you removed the `!= 1.11.0` and the associated comment, but if it's prudent to do so then you should get rid of the trailing space after `coveralls`.

Reply:
I'll find the discussion I had about removing this pin in a different repo.  This change should be back-ported to the skeletons.

Reply:
See cisagov/skeleton-python-library#142.